### ISSUE TITLE
Overlapping hints can occur

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#245]
From https://github.com/phetsims/friction/issues/244#issuecomment-927105540, @terracoda said:

>I also noticed that productive rubbing on a diagonal can trigger both rubbing hints (rub fast or slow and move down to rub harder), though the move down hint seems to get uttered less often. This case is also not ideal after a successful break away.

It should be easy enough to make sure that "move down to rub harder" never occurs if the contactProperty is true. I think that the conditionals in 

https://github.com/phetsims/friction/blob/dbc5402ae3db8a91fa6648464ec85bfe83f91578/js/friction/view/book/BookMovementAlerter.js#L122-L171 

could use some work.

**Comment by zepumph** [comment]
I would like to do a few other issues, then see if this resolves itself. Marking for meeting to make sure I don't forget about it.

**Comment by zepumph** [comment]
I can't really reproduce this. I would love to discuss this during meeting tomorrow. I no longer feel like it has to do with the code linked above (I looked through it and nothing was obvious).

**Comment by zepumph** [comment]
@terracoda said that she was dragging with a mouse diagonally. She said that it was doing it all the time, but right now, on master. This is not occurring. We feel like this is fixed from #244 and https://github.com/phetsims/friction/issues/252. Closing
